src/main/scala/collections/Steps.scala

```diff
@@ -0,0 +1,12 @@
+package collections
```

Comment:
the code should be properly tested. Add unit tests, please

Reply:
added unit tests